custom-recipes/reverse_geocoding/recipe.py

```diff
@@ -1,205 +1,41 @@
 # -*- coding: utf-8 -*-
```

Comment:
I think you could definitely join some parts of codes of the two recipes. The code is almost the same, the only thing changing is the function used, so you could move the code into a new function and take in parameter `revesed=True|False`.